packages/@aws-cdk/aws-scheduler-alpha/lib/group.ts

```diff
@@ -0,0 +1,372 @@
+
+  /**
+   * Emitted when the target returns an exception after EventBridge Scheduler calls the target API.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricTargetErrors(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for invocation failures due to API throttling by the target.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricTargetThrottled(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for dropped invocations when EventBridge Scheduler stops attempting to invoke the target after a schedule's retry policy has been exhausted.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricDropped(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for invocations delivered to the DLQ
+   *
+   * @default - sum over 5 minutes
+   */
+  metricSentToDLQ(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for failed invocations that also failed to deliver to DLQ.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricFailedToBeSentToDLQ(errorCode?: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for delivery of failed invocations to DLQ when the payload of the event sent to the DLQ exceeds the maximum size allowed by Amazon SQS.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricSentToDLQTrunacted(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Grant the indicated permissions on this group to the given principal
+   */
+  grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant;
+  /**
+   * Grant list and get schedule permissions for schedules in this group to the given principal
+   */
+  grantReadSchedules(identity: iam.IGrantable): iam.Grant;
+  /**
+   * Grant create and update schedule permissions for schedules in this group to the given principal
+   */
+  grantWriteSchedules(identity: iam.IGrantable): iam.Grant;
+  /**
+   * Grant delete schedule permission for schedules in this group to the given principal
+   */
+  grantDeleteSchedules(identity: iam.IGrantable): iam.Grant
+}
+
+abstract class GroupBase extends Resource implements IGroup {
+  /**
+   * The name of the schedule group
+   *
+   * @attribute
+   */
+  abstract readonly groupName: string;
+
+  /**
+   * The arn of the schedule group
+   *
+   * @attribute
+   */
+  abstract readonly groupArn: string;
+
+  addSchedules(...schedules: ISchedule[]): void {
+    schedules.forEach(schedule => {
+      // Temporary using ISchedule as parameter for this method,
+      // and checking for Schedule type, until class Schedule is moved from private to public
+      if (schedule instanceof Schedule) {
+        schedule.group = this;
+      }
+    });
+  }
+
+  /**
+   * Return the given named metric for this group schedules
+   *
+   * @default - sum over 5 minutes
+   */
+  metric(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric {
```

Comment:
can we explicitly call out public APIs as `public metric`? Pretty much everywhere -- we prefer to be explicit.